modules/dnn/test/test_int8_layers.cpp

```diff
@@ -974,6 +974,9 @@ TEST_P(Test_Int8_nets, opencv_face_detector)
 
 TEST_P(Test_Int8_nets, EfficientDet)
 {
+    if (cvtest::skipUnstableTests)
+        throw SkipTestException("Skip unstable test");  // detail: https://github.com/opencv/opencv/pull/23167
```

Comment:
Test failed on MSVS2015 build with `--test_threads=3` (also 7, 9):

```
[ RUN      ] Test_Int8_nets.EfficientDet/0, where GetParam() = OCV/CPU
[ INFO:0@94.488] global tf_importer.cpp:3014 cv::dnn::dnn4_v20221220::`anonymous-namespace'::TFImporter::populateNet DNN/TF: parsing model (N/A version info). Number of nodes = 2011
[ INFO:0@94.488] global tf_importer.cpp:3021 cv::dnn::dnn4_v20221220::`anonymous-namespace'::TFImporter::populateNet DNN/TF: parsing config (N/A version info). Number of nodes = 938
Unmatched prediction: class 16 score 0.661760 box [0.226657 x 0.547847 from (0.166423, 0.385289)]
Highest IoU: 0
c:\build\precommit_windows64\4.x\opencv\modules\dnn\test\test_common.impl.hpp(144): error: Value of: matched
  Actual: false
Expected: true
Unmatched reference: class 17 score 0.824592 box [0.244581 x 0.530952 from (0.166575, 0.399682)] IoU diff: 1
c:\build\precommit_windows64\4.x\opencv\modules\dnn\test\test_common.impl.hpp(156): error: Expected: (refScores[i]) <= (confThreshold), actual: 0.824592 vs 0.65
[  FAILED  ] Test_Int8_nets.EfficientDet/0, where GetParam() = OCV/CPU (4345 ms)
```

"Class" is different - no way to tune tolerance.